runtime/System/Console/Hawk/Args/Spec.hs

```diff
@@ -35,8 +35,7 @@ data InputSource
 
 data OutputSink
     = UseStdout
-    -- OutputFile FilePath  -- we might want to implement --in-place
-                            -- in the future
+    | OutputFile FilePath FilePath
```

Comment:
Please add a comment explaining which `FilePath` is which, or even better, create a record whose names clarify their purpose.

Reply:
OK, will clarify in the code: first filepath is the output file, and second filepath is the backup file name in full, if it's null ("") then no backup is created.

Reply:
How about using `Maybe FilePath` for the backup file then